style.css

```diff
@@ -0,0 +1,31 @@
+#main{
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-direction: column;
+  background-color: grey;
+}
+
+#img-div{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80vw;
+  background-color: white;
+}
```

Comment:
Please consider using classes for common attributes instead of separate individual selectors